docs/users/developer-tools.md

```diff
@@ -23,6 +23,18 @@ http://mysite.ddev.local:8025
 
 Please note this will not intercept emails if your application is configured to use SMTP or a 3rd-party ESP integration. If you are using SMTP for outgoing mail handling ([Swiftmailer](https://www.drupal.org/project/swiftmailer) or [SMTP](https://www.drupal.org/project/smtp) modules for example), update your application configuration to use `localhost` on port `1025` as the SMTP server locally in order to use MailHog.
 
+### Database Management
+
+[phpMyAdmin](https://www.phpmyadmin.net/) is a free software tool to manage MySQL and MariaDB databases from a browser. phpMyAdmin comes installed with ddev. 
```

Comment:
Maybe mention that `ddev describe` gets all this info? It's not directly related, but it's a much faster way to find things out.

Reply:
Added a mention to the top of this section on tools.